src/main/openapi/daam.yaml

```diff
@@ -143,14 +143,12 @@ paths:
       responses:
         "204":
           description: Successful Response (no content)
-        "422":
-          description: Validation warnings
+        "400":
+          description: Bad Request
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: "#/components/schemas/ValidationWarnings"
+                  $ref: "#/components/schemas/ErrorResponse"
```

Comment:
important: can you please add errorMessages into `ErrorResponse` payload?

Reply:
I committed the changes according to your comments. you can check now